consensus/hotstuff/votecollector/consensus_verifying_vote_collector.go

```diff
@@ -0,0 +1,16 @@
+package votecollector
+
+import (
+	"sync/atomic"
+
+	"github.com/onflow/flow-go/consensus/hotstuff"
+)
+
+type ConsensusVerifyingVoteCollector struct {
```

Comment:
This seems obsolete we should move members [here](https://github.com/onflow/flow-go/pull/1140/files#diff-9e06d16b426797fa5261aa8342f0e5259eadd121ae65e4bca23774728cbbf852R13)